src/main/permissionsManager.ts

```diff
@@ -239,10 +241,12 @@ t('main.permissionsManager.checkPermission.dialog.message.media');
 t('main.permissionsManager.checkPermission.dialog.message.geolocation');
 t('main.permissionsManager.checkPermission.dialog.message.notifications');
 t('main.permissionsManager.checkPermission.dialog.message.openExternal');
+t('main.permissionsManager.checkPermission.dialog.message.screenShare');
```

Comment:
Will this now show it in the permissions settings window as well?

Reply:
Yep it will